platform/security/sts/security-sts-ldapclaimshandler/src/test/java/ddf/security/sts/claimsHandler/RoleClaimsHandlerTest.java

```diff
@@ -76,6 +77,7 @@ public void testRetrieveClaimsValuesNotNullPrincipal()
     ProcessedClaimCollection processedClaims;
     RoleClaimsHandler claimsHandler;
     SearchResultEntry membershipSearchResult = mock(SearchResultEntry.class);
+    DN resultDN = DN.valueOf("uid=tstark,");
```

Comment:
The base user DN in the test is blank, which causes anything created underneath of it to have a trailing `,` at the end.  The test was created this way originally so I just fit the new changes into it rather than modifying that portion.  I can modify the test to be more realistic in its naming if that makes more sense.

Reply:
Nah. It's fine if that's the way it works. It just reads oddly. Maybe add a comment somewhere near the top so future maintainers aren't confused and think the commas should be removed.